sway-lib-std/src/tx.sw

```diff
@@ -70,9 +72,6 @@ pub enum Transaction {
 ///         Transaction::Script => {
 ///             log("Regular script transaction");
 ///         },
-///         Transaction::Create => {
-///             log("Contract deployment transaction");
-///         },
```

Comment:
`Create` still exists as a variant, so why is this removed?

Reply:
I think it's because the only time a creation tx would be running any sway code would be inside of predicates, and predicates can't log.